crates/optimism/rpc/src/sequencer.rs

```diff
@@ -7,8 +7,38 @@ use alloy_primitives::hex;
 use alloy_rpc_client::{ClientBuilder, RpcClient as Client};
 use alloy_rpc_types_eth::erc4337::TransactionConditional;
 use alloy_transport_http::Http;
+use thiserror::Error;
 use tracing::warn;
 
+type Result<T, E = Error> = std::result::Result<T, E>;
```

Comment:
we dont need this alias, I prefer to be explicit when using the error here, because then this is obvious what kind of result is being returned.